config/cars/mods/urd/urd_t5_bayro_2018.ini

```diff
@@ -0,0 +1,96 @@
+[INCLUDE]
+INCLUDE=common/no_popup_lights.ini
```

Comment:
This is included by default, there's no need to redefine it unless there's an error in guessing.

Reply:
probably also copied from one of the examples (e.g. ks_maserati_quattroporte.ini)